### PR TITLE
Recipient contact person in excel

### PR DESCRIPTION
**Pull request by SindreVatnaland**
Person which submitted a family is added to the excel file

excel

### CODE REVIEW

**Comment by SindreVatnaland** [comment]
#583

File: ClientApp/src/components/admin/common/RecipientTable.tsx [hunk 1]
```diff
@@ -102,7 +102,10 @@ const DatatableRecipient: React.FC<Props> = ({
             id="panel1bh-header"
           >
             <Typography className={classes.heading}>
-              {recipient.familyId}
+            <Typography className={classes.heading}>
```
Comment (Sommerli) [inline ClientApp/src/components/admin/common/RecipientTable.tsx:105]: Duplicate Typography

**Review by TTorgersen — DISMISSED** [review]
There is a merge conflict, other than that looks good.

**Review by TTorgersen — DISMISSED** [review]

**Review by Sommerli — APPROVED** [review]

File: GiEnJul/Infrastructure/AutoMapperProfile.cs [hunk 2]
```diff
@@ -93,7 +96,8 @@ public AutoMapperProfile()
 
             //Connection mapping
             CreateMap<Entities.Connection, Utilities.ExcelClasses.DeliveryExcel>()
-                .ForMember(dest => dest.Check, opt => opt.Ignore());
+                .ForMember(dest => dest.Check, opt => opt.Ignore())
+                .ForMember(dest => dest.SubmitterMail, opt => opt.Ignore());
```
Comment (Sommerli) [inline GiEnJul/Infrastructure/AutoMapperProfile.cs:100]:
```suggestion
                .ForMember(dest => dest.SubmitterMail, src => src.ContactEmail);
```

Comment (Sommerli) [inline GiEnJul/Infrastructure/AutoMapperProfile.cs:100]:
```suggestion
                
```
Delete this

File: GiEnJul/Utilities/ExcelEntries/DeliveryExcel.cs [hunk 3]
```diff
@@ -10,6 +10,8 @@ public class DeliveryExcel : IExcel
         public string ReferenceId { get; set; }
         public string GiverFullName { get; set; }
         public string GiverPhoneNumber { get; set; }
+        public string SubmitterFullName { get; set; }
+        public string SubmitterMail { get; set; }
```
Comment (Sommerli) [inline GiEnJul/Utilities/ExcelEntries/DeliveryExcel.cs:14]:
```suggestion
        public string SubmitterEMail { get; set; }
```

File: GiEnJul/Utilities/ExcelEntries/DeliveryExcel.cs [hunk 4]
```diff
@@ -21,6 +23,8 @@ public OrderedDictionary AsOrderedDictionary()
                 ["Referanse id"] = ReferenceId,
                 ["Giver navn"] = GiverFullName,
                 ["Giver mobil"] = GiverPhoneNumber,
+                ["Kontaktperson"] = SubmitterFullName,
+                ["Kontaktperson mail"] = SubmitterMail,
```
Comment (Sommerli) [inline GiEnJul/Utilities/ExcelEntries/DeliveryExcel.cs:27]:
```suggestion
                ["Kontaktperson mail"] = SubmitterEMail, 
```

**Review by Sommerli — CHANGES_REQUESTED** [review]
Automapper has had some problems with mail-property, as it did not reflect the Connection.SubmitterEmail field properly. missing an E

**Review by Sommerli — DISMISSED** [review]

File: ClientApp/src/common/components/Types.tsx [hunk 5]
```diff
@@ -17,6 +17,7 @@ export interface SelectedConnectionType {
     partitionKey: string;
     rowKey: string;
     phoneNumber: string;
+    matchedRecipientId: string;
```
Comment (Sommerli) [inline ClientApp/src/common/components/Types.tsx:20]: matchedRecipientId could be changed to matchedFamilyId

Comment (SindreVatnaland) [reply ClientApp/src/common/components/Types.tsx]: This part of the code is used and fixed in another approved PR..
Removed this part of the code

**Review by TTorgersen — APPROVED** [review]

**Pull request merged**